CICD.Tools.SDKChecker/Program.cs

```diff
@@ -47,73 +46,53 @@ public static async Task<int> Main(string[] args)
                 IsRequired = false
             };
 
+            var solutionFilePath = new Option<string>(
+                name: "--solution-filepath",
+                description: "The filepath to the solution.");
```

Comment:
Would indicate this as Optional, for developer clarity IsRequired = false.

Reply:
Done